CodeStarterCamp_Week2/main.swift

```diff
@@ -6,3 +6,78 @@
 //  Copyright © yagom academy. All rights reserved.
 //
 
+import Foundation
+
+let lotto: Set<Int> = getlotto()
+
+func getlotto() -> Set<Int>{
+    
+    var lotto: Set<Int> = Set<Int>()
+    
+    repeat {
+        
+        lotto.insert(Int.random(in: (1...45)))
+        
+    } while (lotto.count < 6)
+    
+    return lotto
+    
+}
+
+
+
+func getIntersectionLottoNumber() -> [Int]{
+    
+    let myLottoNumbers: [Int] = [1, 2, 3, 4, 5, 6]
+    var lottoNumber: Set<Int> = Set<Int> (myLottoNumbers)
```

Comment:
```suggestion
    var lottoNumber: Set<Int> = Set(myLottoNumbers)
```

요소의 타입이 같을 경우 `Set`, `Array`처럼 타입 이름만 명시해주어도 괜찮습니다!